src/CronJobListBuilder.php

```diff
@@ -57,7 +57,15 @@ public function buildRow(EntityInterface $entity) {
     $row['started'] = $entry->start_time ? \Drupal::service('date.formatter')->format($entry->start_time, "short") : $this->t('Never');
     // In milliseconds.
     $row['duration'] = round(($entry->end_time - $entry->start_time) * 1000, 0);
-    $row['status'] = $entity->status() ? $this->t('Enabled') : $this->t('Disabled');
+    if ($entity->isValid() && !$entity->status()) {
```

Comment:
I think this logic is easier to implement if you check valid first. if (valid) missing elseif (enabled) disabled else enabled.